packages/supabase_flutter/README.md

```diff
@@ -41,108 +41,186 @@ void main() async {
 final supabase = Supabase.instance.client;
 ```
 
-> `authCallbackUrlHostname` is optional. It will be used to filter Supabase authentication redirect deeplink. You need to provide this param if you use deeplink for other features on the app.
-
-> `debug` is optional. It's enabled by default if you're running the app in debug mode (`flutter run --debug`).
-
 ## Usage example
 
 ### [Authentication](https://supabase.com/docs/guides/auth)
 
 ```dart
-import 'package:supabase_flutter/supabase_flutter.dart';
-
 final supabase = Supabase.instance.client;
 
-class MyWidget extends StatefulWidget {
-  const MyWidget({Key? key}) : super(key: key);
+// Email and password sign up
+await supabase.auth.signUp(
+  email: email,
+  password: password,
+);
 
-  @override
-  State<MyWidget> createState() => _MyWidgetState();
-}
+// Email and password login
+await supabase.auth.signInWithPassword(
+  email: email,
+  password: password,
+);
 
-class _MyWidgetState extends State<MyWidget> {
-  late final StreamSubscription<AuthState> _authSubscription;
-  User? _user;
+// Magic link login
+await supabase.auth.signInWithOtp(email: 'my_email@example.com');
 
-  @override
-  void initState() {
-    _authSubscription = supabase.auth.onAuthStateChange.listen((data) {
-      final AuthChangeEvent event = data.event;
-      final Session? session = data.session;
-      setState(() {
-        _user = session?.user;
-      });
-    });
-    super.initState();
-  }
+// Listen to auth state changes
+supabase.auth.onAuthStateChange.listen((data) {
+  final AuthChangeEvent event = data.event;
+  final Session? session = data.session;
+  // Do something when there is an auth event
+});
+```
 
-  @override
-  void dispose() {
-    _authSubscription.cancel();
-    super.dispose();
-  }
+#### Native Apple Sign in
```

Comment:
@Vinzent03 
The `signInWithApple()` instructions were not removed in this PR, but moved here!

Reply:
Oh sorry must have overseen this.